go/libraries/doltcore/mvdata/table_data_loc.go

```diff
@@ -203,14 +212,14 @@ type tableEditorWriteCloser struct {
 	initialData types.Map
 	opsSoFar    int64
 	statsCB     noms.StatsCB
-	tableEditor *doltdb.TableEditor
+	tableEditor *doltdb.SessionedTableEditor
 	tableSch    schema.Schema
 }
 
 var _ DataMoverCloser = (*tableEditorWriteCloser)(nil)
 
-func (te *tableEditorWriteCloser) GetTable(ctx context.Context) (*doltdb.Table, error) {
-	return te.tableEditor.Table()
+func (te *tableEditorWriteCloser) GetRoot(ctx context.Context) (*doltdb.RootValue, error) {
```

Comment:
Same comment as on table editor. This does write IO, should not be called Get*. Should call this CommitChanges or similar.